oeq-ts-rest-api/src/Utils.ts

```diff
@@ -0,0 +1,40 @@
+import * as Lodash from "lodash";
```

Comment:
Could we avoid importing all of lodash for one function?

Instead using

```suggestion
import { cloneDeep } from "lodash";
```

with https://github.com/lodash/babel-plugin-lodash

or
```suggestion
import * as cloneDeep from "lodash/cloneDeep";
```